modules/ROOT/pages/_partials/mule-runtime-metrics.adoc

```diff
@@ -46,41 +46,62 @@ The maximum number of concurrent flows across all Mule apps and business groups
 // end::cards[]
 
 // tag::table[]
-Business Group:: 
-The business group in which a Mule app is deployed.
 
-Environment:: 
-The environment in which a Mule app is deployed.
+Select the *Business Group Details* tab to view:
 
-Application::
-The name of the Mule application.
+[%header%autowidth.spread]
+|===
+|Field | Description
+
+|Business Group |The business group a Mule app is deployed to
+
+|Environment Type |The environment in which a Mule app is deployed.
+
+| Cluster |The name of the cluster containing the replicas and worker nodes.
+
+|Cluster Capacity (Millicores) | The worker node capacity.
+|===
+
+Select the *Application Details* tab to view:
+
+[%header%autowidth.spread]
+|===
+|Field |Description
+|Application |The name of the Mule application.
 
-Deployment:: 
-The runtime plane to which the Mule app is deployed. CloudHub (abbreviated as CH), CloudHub 2.0 (abbreviated as CH2), and Runtime Fabric (abbreviated as RTF) are supported. 
+| Business Group | The business group a Mule app is deployed to.
 
-Flows:: 
-The sum of flows in the Mule app. This number is calculated by multiplying flows by the number of workers (CloudHub) or replicas (CloudHub 2.0). For more information about how this metric is billed, see xref:usage-metrics.adoc#mule-flows[Mule flows].
+| Deployment Type | The runtime plane to which the Mule app is deployed. CloudHub (abbreviated as CH), CloudHub 2.0 (abbreviated as CH2), and Runtime Fabric (abbreviated as RTF) are supported. 
 
-Messages::
-The sum of inbound and outbound Mule messages in the Mule app. For more information about how this metric is billed, see xref:usage-metrics.adoc#mule-messages[Mule messages].  
+|Environment Type |The environment in which a Mule app is deployed.
 
-Data Throughput::
-The sum of inbound and outbound data in gigabytes (GB) transmitted by the Mule app. For more information about how this metric is billed, see xref:usage-metrics.adoc#data-throughput[data throughput].  
+| Cluster | The name of the cluster containing the replicas and worker nodes.
+
+| CPU Limit | The maximum amount of vCPU the application can use. This is shared CPU on the worker node.
+
+| CPU Reserve (Millicores) | The amount of vCPU guaranteed to the application and reserved for its use.
+
+| Mule Flows | The sum of flows in the Mule app. This number is calculated by multiplying flows by the number of workers (CloudHub) or replicas (CloudHub 2.0). For more information about how this metric is billed, see xref:usage-metrics.adoc#mule-flows[Mule flows].
+
+| Mule Messages | The sum of inbound and outbound Mule messages in the Mule app. For more information about how this metric is billed, see xref:usage-metrics.adoc#mule-messages[Mule messages].  
```

Comment:
```suggestion
| Mule Messages | The sum of inbound and outbound Mule messages in the Mule app. For more information about how this metric is billed, see xref:usage-metrics.adoc#mule-messages[Mule Messages].  
```

Reply:
maybe lc is right, though?

Reply:
It seems weird.

Reply:
I don't know. It's not capitalized in the legacy doc.